vm/src/vm.rs

```diff
@@ -303,33 +303,26 @@ impl VirtualMachine {
     }
 
     pub fn import(&self, module: &str, from_list: &PyObjectRef, level: usize) -> PyResult {
-        let sys_modules = self
-            .get_attribute(self.sys_module.clone(), "modules")
-            .unwrap();
-        if let Ok(module) = sys_modules.get_item(module.to_string(), self) {
-            Ok(module)
-        } else {
-            match self.get_attribute(self.builtins.clone(), "__import__") {
-                Ok(func) => self.invoke(
-                    func,
-                    vec![
-                        self.ctx.new_str(module.to_string()),
-                        if self.current_frame().is_some() {
-                            self.get_locals().into_object()
-                        } else {
-                            self.get_none()
-                        },
-                        self.get_none(),
-                        from_list.clone(),
-                        self.ctx.new_int(level),
-                    ],
-                ),
-                Err(_) => Err(self.new_exception(
-                    self.ctx.exceptions.import_error.clone(),
-                    "__import__ not found".to_string(),
-                )),
-            }
-        }
+        let sys_modules = self.get_attribute(self.sys_module.clone(), "modules")?;
+        sys_modules.get_item(module.to_string(), self).or_else(|_| {
```

Comment:
Why did you change the original code to a lambda function with the `or_else` statement? Is there a performance benefit of using a lambda function? Personally I find the code a little harder to understand when a lot of lambda functions are involved.

Reply:
I personally find it more readable because I think the intent is clearer. "Get it from `sys.modules`, or else if that fails try calling the import function.

Reply:
Hmm, I guess it is a matter of personal preference/opinion? Maybe we could settle on having both ways, and leaving the code in either style when touching the code? Does that sound good to you? I'll do my best on this strategy at least.